tests/integration/store/rollback_test.go

```diff
@@ -4,26 +4,63 @@ import (
 	"fmt"
 	"testing"
 
-	"cosmossdk.io/simapp"
 	dbm "github.com/cosmos/cosmos-db"
 	abci "github.com/tendermint/tendermint/abci/types"
-	"github.com/tendermint/tendermint/libs/log"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 	"gotest.tools/v3/assert"
 
+	"github.com/cosmos/cosmos-sdk/runtime"
+	"github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/testutil/configurator"
 	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
+
+	_ "github.com/cosmos/cosmos-sdk/x/auth"
```

Comment:
to be deleted. I think the configurator could be the place where these modules are configured.
However, then we have the configurator depending on modules which means the sdk as well (when they are all in go.mod).